server/pyproject.toml

```diff
@@ -6,12 +6,14 @@ authors = ["James Collier <james.collier@vib.be>"]
 
 [tool.poetry.dependencies]
 python = "^3.9"
-fastapi = "^0.70.0"
```

Comment:
Probably shouldn't be a downgrade

Reply:
### Issue n°2
As far as I know, there was no reason to downgrade the version of fastapi.
### Issue n°3
The two database revisions were successfully merged without any compiler errors.


Resolved these issues in latest commit (e0f8cc2).